### DISCUSSION TITLE
Alesis QuadraSynth panel for ctrlr

### DISCUSSION THREAD

**Discussion by troutstudio** [RomanKubiak/ctrlr#549] [Ideas]
Would this be very difficult? I'd contribute to this in any way I could. I own one of course but I don't know how difficult the task is.

**Comment by dnaldoog** [comment]
Hi @troutstudio 

I'll bet it is difficult! Most panels are.

1. Best thing is to start simple (ignore sysex dump and file save functionality). 
2. Copy someone else's panel.
3. If possible, download Sound Diver and see how they do it by opening the MIDI monitor and watching the different messages. The Alesis QuadraSynth is listed.
4. If you run into trouble post your question(s) here!
